modules/ROOT/partials/configuration/dialog_streamContent.adoc

```diff
@@ -0,0 +1,58 @@
+[[streamContent]]
+== `streamContent`
+
+`+streamContent+` is an optional property, available to add to the `+iframe+` dialog component.
+
+The property has a default value of `+false+`.
+
+When `+streamContent+` is set to true, content presented within the `+iframe+` dialog component is updated using the `document.write()` method, rather than the by using the `+srcdoc+` attribute.
+
+The `+srcdoc+` attribute requires the `+iframe+` be reloaded each time the content changes. This results in a flickering effect.
+
+As well, performing updates via the `+srcdoc+` attribute means the contents of the `+iframe+` are not accessible. As a consequence, it is not possible to automatically scroll the `+iframe+` contents to the bottom as it gets updated.
+
+Setting `+streamContent+` to `+true+`, and switching the update mechanism to the `document.write()` method, stops the flickering and allows the `+iframe+` to constantly present the incoming content at an end-of-the-current data position.
```

Comment:
@MitchCTiny Is it worth mentioning the `string concatenation` here?

Reply:
No, since this is documentation for the core change. String concatenation is something we do in the AI plugin to keep track of the overall AI response, but this doesn't always need to happen depending on the type of usage.